server/routes/index.ts

```diff
@@ -63,15 +53,42 @@ export default function routes(service: Services): Router {
     const { user } = res.locals
     const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
     const incidentReportingApi = new IncidentReportingApi(systemToken)
+    const manageUsersApiClient = new ManageUsersApiClient()
 
     // No authorisation at this point, no data shown in production
     if (config.environment === 'prod') {
       throw new NotFound()
     }
 
     const report = await incidentReportingApi.getReportWithDetailsById(id)
+    const reportedBy = (await manageUsersApiClient.getNamedUser(systemToken, report.reportedBy))?.name
+
+    res.render('pages/report', { report, reportedBy })
+  })
+
+  get('/incidents', async (req, res, next) => {
+    const { user } = res.locals
+    const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
+    const incidentReportingApi = new IncidentReportingApi(systemToken)
+    const manageUsersApiClient = new ManageUsersApiClient()
+
+    const incidents = (
+      await incidentReportingApi.getEvents({
+        // prisonId: user.activeCaseLoadId,
+        // eventDateFrom: new Date('2024-07-30'),
+        // eventDateUntil: new Date('2024-07-30'),
+        // sort: ['eventDateAndTime,ASC'],
+      })
+    )?.content
+
+    const inputUsernames = []
+    for (const incident of incidents) {
+      inputUsernames.push(incident.modifiedBy)
+    }
```

Comment:
This could be _something like_ (haven't tested but should be it): 

```JavaScript
const usernames = incidents.map(incident => incident.modifiedBy)
```

Reply:
replaced these now